Source/WebCore/dom/wpe/PointerEventWPE.cpp

```diff
@@ -71,9 +71,14 @@ Ref<PointerEvent> PointerEvent::create(const AtomString& type, const PlatformTou
     return adoptRef(*new PointerEvent(type, event, coalescedEvents, predictedEvents, typeCanBubble(type), typeIsCancelable(type), index, isPrimary, WTFMove(view), touchDelta));
 }
 
+// According to the PointerEvents spec all active pointer ids have to be unique.
+// Libinput on Linux assigns the ids of the touchpoints starting at 0, but
+// the ids 0 and 1 are used for the pointer ids of mouse and pen/stylus.
+const unsigned touchMinimumPointerId = WebCore::mousePointerID + 1;
```

Comment:
`WebCore::mousePointerID` is a static constexpr, so maybe this can be made one, too?